pawn/Elements/AdministratorCommands.pwn

```diff
@@ -972,6 +972,11 @@ lvp_show(playerId, params[]) {
         showInfo = true;
         format(g_message, sizeof(g_message), "Save money and stats! Register your nickname at www.sa-mp.nl");
     }
+    
+    else if (!strcmp(showParameter, "discord", true, 7)) {
```

Comment:
@RussellLVP , updated here as well as in /guide request too . Feel free to use that as it has these script lines too